pkg/monitor/sqsevent/sqs-monitor.go

```diff
@@ -101,6 +101,19 @@ func (m SQSMonitor) processSQSMessage(message *sqs.Message) (*EventBridgeEvent,
 	event := EventBridgeEvent{}
 	err := json.Unmarshal([]byte(*message.Body), &event)
 
+	if err == nil && len(event.DetailType) == 0 && len(event.Time) > 0 {
+		log.Info().Msg("no EventBridge detail-type found, assuming a Lifecycle termination event")
```

Comment:
why is `len(event.Time) > 0` also required?

Reply:
Originally added since it is a common field between EventBridge and Lifecycle events, and caused some confusing behavior with handling EventBridge events with default values. However, after reviewing again, I'll remove this in favor of more explicit checks for Lifecycle events.